netbox_metatype_importer/views.py

```diff
@@ -209,9 +211,163 @@ class MetaModuleTypeImportView(GenericTypeImportView):
 
 class MetaDeviceTypeBulkDeleteView(generic.BulkDeleteView):
     queryset = MetaType.objects.filter(type=TypeChoices.TYPE_DEVICE)
+    filterset = MetaTypeFilterSet
     table = MetaTypeTable
 
+    def post(self, request, **kwargs):
```

Comment:
Why was the method override required here?

Reply:
By mistake it was added, Now i have removed it